examples/ignition/k8s-worker.yaml

```diff
@@ -58,7 +58,7 @@ systemd:
           --register-node=true \
           --allow-privileged=true \
           --config=/etc/kubernetes/manifests \
-          --hostname-override={{.ipv4_address}} \
+          --hostname-override={{.domain_name}} \
```

Comment:
Any name which resolves to the host can be used here. We discussed some variable name choices (including `name`, `dns_name`), but decided it was better to go with `domain_name`. `name` is too broad and `dns_name` implied the metadata value was used for DNS configuration when it is not.